build/circle-test.sh

```diff
@@ -49,14 +51,14 @@ fi
 # better.
 if [ "${CIRCLE_TEST_REPORTS}" != "" ]; then
     if [ -f "${outdir}/test.log" ]; then
-	mkdir -p "${CIRCLE_TEST_REPORTS}/go"
-	${builder} go2xunit < "${outdir}/test.log" \
-	       > "${CIRCLE_TEST_REPORTS}/go/test.xml"
+    mkdir -p "${CIRCLE_TEST_REPORTS}/go"
```

Comment:
Yes, a tab is a tab, but in this file (and in pretty much everything but Go source files), the intended interpretation of a tab character is 8 spaces. This file was using four spaces for a single indent and one tab for a double indent.